m/mod_wsgi/mod_wsgi_CENTOS8.sh

```diff
@@ -0,0 +1,37 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : mod_wsgi
+# Version       : 4.7.1
+# Source repo   : https://github.com/GrahamDumpleton/mod_wsgi
+# Tested on     : RHEL 7.6, RHEL 7.7
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vedang Wartikar <vedang.wartikar@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+export PYTHON=python3
+export PIP=pip3
+export REPO=https://github.com/GrahamDumpleton/mod_wsgi
+
+yum update -y
+
+# install python
+yum install ${PYTHON} ${PYTHON}-devel ${PYTHON}-pip -y
+
+# install mod_wsgi and related dependencies
+yum install ${PYTHON}-mod_wsgi httpd httpd-devel gcc redhat-rpm-config -y
```

Comment:
I dont think we need **redhat-rpm-config** package. Please check and remove

Reply:
The build is failing, if not installed, with error message
```
  gcc: error: /usr/lib/rpm/redhat/redhat-hardened-cc1: No such file or directory
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for mod-wsgi
```